rosco-web/config/packer/install_packages.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -e # Make rush fail the build
```

Comment:
Please move the comment to the preceding line and terminate it with a period (trying to match the other comments in this script).